src/jdk.jcmd/share/man/jcmd.1

```diff
@@ -35,7 +35,7 @@
 . ftr VB CB
 . ftr VBI CBI
 .\}
-.TH "JCMD" "1" "2023" "JDK 21-ea" "JDK Commands"
```

Comment:
Sorry my bad I gave you the wrong command to use to generate this. I should have been VERSION_SHORT=21-ea

Reply:
No worries, I'll just revert the change.